fmriprep_rodents/workflows/bold/confounds.py

```diff
@@ -188,7 +188,7 @@ def init_bold_confs_wf(
     # Get masks ready in T1w space
     acc_tpm = pe.Node(AddTPMs(indices=[1, 2]),  # BIDS convention (WM=1, CSF=2)
                       name='acc_tpm')  # acc stands for aCompCor
-    csf_roi = pe.Node(TPM2ROI(erode_mm=0, mask_erode_mm=30), name='csf_roi')
+    csf_roi = pe.Node(TPM2ROI(erode_mm=0, mask_erode_mm=3), name='csf_roi')
```

Comment:
I think you can go a bit lower in this case.

Reply:
Lower as in make the kernel smaller, or lower like more erosion?